_docs/instructor/autograding/jupyter_notebook.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Jupyter Notebooks
+category: Instructor > Autograding
+---
+
+### Example Configuration
+
+The following configuration can be found on the [Submitty GitHub](https://github.com/Submitty/Submitty/tree/main/more_autograding_examples/jupyter_notebook_autograding).
+
+```json
```

Comment:
please don't include the full json here - this will be out of date very quickly when we make any edits to the example